src/AriesCounters.c

```diff
@@ -37,6 +37,27 @@ char *caller_program;
 /* utilities for timing each region/timestep */
 unsigned long long tempo1, tempo2;
 
+struct timeval startTime;
+
+struct timeval get_timestamp() {
```

Comment:
We could unify this with the get_time_ns call so that we call gettimeofday only once before and once after. We could save startTime (the entire struct) in StartSysTimer and then calculate elapsedTime in EndRecordAriesCounters itself.